SimpleLottery/Model/Lottery.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Lottery.swift
+//  SimpleLottery
+//
+//  Created by HyungJung Kim on 18/03/2019.
+//  Copyright © 2019 HyungJung Kim. All rights reserved.
+//
+
+import Foundation
+
+struct Lottery {
+    
+    static let price = 1000
+    static let numbersRange = Range(1 ... 45)
```

Comment:
```suggestion
    static let numbersRange = 1 ... 45
``` 

`Range`와 괄호를 생략하고 `ClosedRange`로 선언할  수 있을 것 같습니다.

https://medium.com/@lucianoalmeida1/exploring-range-types-in-swift-e0e7ab27ab79